core/src/translate/expr.rs

```diff
@@ -1,3 +1,5 @@
+use sqlparser::ast::Array;
+
 use {
```

Comment:
could you merge this to use single `use`?

Reply:
oh, i will apply this ASAP